services/shhext/publisher/service.go

```diff
@@ -69,17 +70,20 @@ func New(config *Config, w *whisper.Whisper) *Service {
 }
 
 // InitProtocolWithPassword creates an instance of ProtocolService given an address and password used to generate an encryption key.
+// TODO(adam): replace with SetPFSProtocolService()
```

Comment:
as a side note, we should stop calling the protocol `PFS` as it handles encryption/multidevices etc, and some messages have no PFS (DH), so we should probably move away from the naming